schema/types/isolator_linux_specific_test.go

```diff
@@ -204,3 +204,40 @@ func TestNewLinuxSeccompRetainSet(t *testing.T) {
 		}
 	}
 }
+func TestLinuxOOMScoreAdj(t *testing.T) {
+	tests := []struct {
+		in []string
+
+		wset []LinuxCapability
```

Comment:
The codepath tested here doesn't exist for the OOM case, because it's a scalar / simple value.